tests/GridFS/BucketFunctionalTest.php

```diff
@@ -17,6 +17,9 @@
  */
 class BucketFunctionalTest extends FunctionalTestCase
 {
+    /*
```

Comment:
This is missing a second `*` in order to be considered a doc block. Without it, [`ReflectionFunction::getDocComment()`](http://php.net/manual/en/reflectionfunctionabstract.getdoccomment.php) is not able to pick it up for parsing by PHPUnit.